src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -737,20 +735,23 @@ public List<EventSummary> getEventsForProject(final Integer projectId, final Str
     return events.stream().map(event -> {
 
       TenderStatus statusCode;
-
+      RfxSetting rfxSetting=null;
       if (event.getExternalEventId() == null) {
         var assessment = assessmentService.getAssessment(event.getAssessmentId(), Optional.empty());
         statusCode = TenderStatus.fromValue(assessment.getStatus().toString().toLowerCase());
       } else {
-        var exportRfxResponse = jaggaerService.getRfx(event.getExternalEventId());
-        statusCode = jaggaerAPIConfig.getRfxStatusToTenderStatus()
+         var exportRfxResponse = jaggaerService.getRfx(event.getExternalEventId());
+         statusCode = jaggaerAPIConfig.getRfxStatusToTenderStatus()
             .get(exportRfxResponse.getRfxSetting().getStatusCode());
+         rfxSetting=exportRfxResponse.getRfxSetting();
```

Comment:
Formatter settings might be a bit off - we use GoogleStyle, generally speaking (should be plugin for Eclipse & IntelliJ - Venki has details for the latter 👍